### PR TITLE
Range bot improvements

### PR DESCRIPTION
**Pull request by RabidPope**
Changes for the hoplite and mongoose.
Increased turret traversal and decreased turn speed so the units don't stop firing when making sharp turns.
The mongoose has a 2 second firing delay for some reason.
The hoplite turret seems to be unable to stay on target unless yaw speed is increased significantly higher than turn speed.
Also slightly increased hoplite projectile velocity as even very slow units such as coms can easily dodge.

### CODE REVIEW

**Comment by Garanas** [comment]
Better 👍 , thank you for rebasing it

**Comment by Garanas** [comment]
@Tagada14 all yours

**Comment by RabidPope** [comment]
@Garanas @Tagada14 Would it be reasonable to reduce the mongoose weapon charge time?

**Comment by Garanas** [comment]
You'd need the animation to match. Reducing it to `0.1` is not desireable, but lowering it from `2` to `1` and then speeding up the animation seems fine to me.

**Comment by Tagada14** [comment]
We are discussing proposed changes, they look pretty good to me.

**Comment by RabidPope** [comment]
@Tagada14 So any feedback on if this is desirable?

**Comment by MrRowey** [comment]
@RabidPope were active getting thought the backlog this will be brought up on this Sundays meeting Sorry for this take so long

**Comment by MrRowey** [comment]
@RabidPope we agree with this but we would prefer the `MuzzleVelocityRandom = 0.5,` for the mongoose

File: units/DEL0204/DEL0204_unit.bp [hunk 1]
```diff
@@ -328,7 +328,7 @@ UnitBlueprint {
             MuzzleSalvoDelay = 0.3,
             MuzzleSalvoSize = 4,
             MuzzleVelocity = 12.8,
-            MuzzleVelocityRandomness = 0.5,
+            MuzzleVelocityRandom = 0.5,
```
Comment (Garanas) [inline units/DEL0204/DEL0204_unit.bp:331]: The name of the field is different. Is this a typo?

Comment (RabidPope) [reply units/DEL0204/DEL0204_unit.bp]: Just changed it

Comment (RabidPope) [reply units/DEL0204/DEL0204_unit.bp]: Its "random" on live and "randomness" on beta. Do I just keep it as randomness?

**Comment by ComradeStryker** [comment]
Should the ACU not also get this change?
They suffer from the same issue.

**Pull request merged**